rendered/manifests/eks-fargate/clusterRole.yaml

```diff
@@ -15,28 +15,6 @@ metadata:
     release: default
     heritage: Helm
 rules:
-- apiGroups:
```

Comment:
Ideally only functional content changes would alter how the manifests are rendered.

Reply:
Ok, I brought the old way back and it detects no differences now